tiered-storage/file-system/src/main/java/org/apache/bookkeeper/mledger/offload/filesystem/impl/FileSystemManagedLedgerOffloader.java

```diff
@@ -208,7 +225,10 @@ public void run() {
                     long end = Math.min(needToOffloadFirstEntryNumber + ENTRIES_PER_READ - 1,
                             readHandle.getLastAddConfirmed());
                     log.debug("read ledger entries. start: {}, end: {}", needToOffloadFirstEntryNumber, end);
+                    long startReadTime = System.nanoTime();
                     LedgerEntries ledgerEntriesOnce = readHandle.readAsync(needToOffloadFirstEntryNumber, end).get();
+                    this.mxBean.recordReadLedgerLatency(extraMetadata.get(MANAGED_LEDGER_NAME),
```

Comment:
Please use `topicName` to store the value of `extraMetadata.get(MANAGED_LEDGER_NAME)`, avoiding call it multiple times.